libraries.json

```diff
@@ -338,5 +338,11 @@
     "github": "msink/kotlin-libui",
     "category": "UI",
     "maven": "https://dl.bintray.com/msink/kotlin-native/com/github/msink/libui/"
+  },
+
+  {
+    "github": "robstoll/atrium",
+    "category": "Test",
+    "maven": "https://dl.bintray.com/robstoll/tutteli-jars/ch/tutteli/atrium/"
```

Comment:
here no artifacts with gradle metadata as i see. now autofetch can't get info without gradle metadata.
here example of gradle metadata - https://dl.bintray.com/msink/kotlin-native/com/github/msink/libui/0.1.7/libui-0.1.7.module

Reply:
maybe you have some artifacts with gradle metadata?

Reply:
Oh... something must be wrong then, I'll close the PR and re-open a new one once I have gradle meta-data in it.